.dotnet/api/OpenAI.netstandard2.0.cs

```diff
@@ -1401,10 +1412,27 @@ public class ChatOutputAudioReference : IJsonModel<ChatOutputAudioReference>, IP
     }
     public class ChatOutputTokenUsageDetails : IJsonModel<ChatOutputTokenUsageDetails>, IPersistableModel<ChatOutputTokenUsageDetails> {
         public int AudioTokenCount { get; }
+        public int PredictionAcceptedTokenCount { get; }
+        public int PredictionRejectedTokenCount { get; }
```

Comment:
Should we call these "AcceptedPredictionTokenCount" and "RejectedPredictionTokenCount"?

Pros:
1. It reads more naturally.
2. It aligns with the REST API.

Cons:
1. They won't appear next to each other when sorted alphabetically (e.g., in Intellisense).